src/app/(main)/myprofile/[userId]/page.tsx

```diff
@@ -0,0 +1,55 @@
+import NoList from '@/src/components/applyList/NoList';
+import MyProfile from '@/src/components/myProfile/MyProfile';
+import ApplyTable from '@/src/components/applyList/ApplyTable';
+import getProfileData from '@/src/api/getProfileData';
+import getUserApplyCount from '@/src/api/getUserApply';
+import styles from './page.module.scss';
+
+interface MyPageProps {
+  params: {
+    [param: string]: string;
+  };
+}
+// const userType = 'employee' : employee일 때 해당 페이지로 이동
+
+async function mypage({ params }: MyPageProps) {
+  const { userId } = params;
+
+  const result = await getProfileData(userId);
+  const { name, phone, address, bio } = result;
```

Comment:
이 부분은 이렇게 줄일 수 있을 것 같네요~!! 
```suggestion
  const { name, phone, address, bio } = await getProfileData(userId);
```